STM32_Mini12864/klipper-mini12864.cfg

```diff
@@ -1,6 +1,7 @@
 [mcu menu]
 # Uncomment serial and update with proper path
-#serial: /dev/ttyACM0
+# Find serial path by running terminal command: ls /dev/serial/by-id/*
+#serial: /dev/serial/by-id/usb-Klipper_stm32f042x6_
```

Comment:
Might be best to comment OUT this line and remove the default serial so that the end user is FORCED to verify that the serial port is their own.

Reply:
I'm using `#serial: **PASTE YOUR SERIAL PORT HERE AND UNCOMMENT**` for the V0 display now to not cause ANY issues with people not realizing they need to update the port

Reply:
Good idea.  I'll make the change.